### DISCUSSION TITLE
[Windows] Issues creating VM from custom image

### DISCUSSION THREAD

**Discussion by Reterino** [actions/virtual-environments#11146] [General questions]
Hey all,

I've been creating an image using the Packer build, this works all the way to the end and successfully syspreps the VM to make an image. Packer reports no issue, and the image is created without issue.

I then go to create a test VM from the image and it gets stuck in "Status: Creating" which eventually times out. But I can go log in to the VM with the admin account and use it without issue. This isn't great though, as setting up the agent as a scale set requires the VM Agent to be working to install the DevOps extension.

The boot diagnostics screenshot is set on a black screen during creation and the serial console log provides no errors. Sysprep log also has no errors (attached).
[setupact.log](https://github.com/user-attachments/files/18087250/setupact.log)

Does anyone have any advice on how to fix this issue?

Thanks,
Reterino

**Comment by Reterino** [comment]
I've found that the services for RdAgent and Windows Azure Guest Agent are both set to Disabled, is that something sysprep/this image process is doing?

**Comment by lallen-0987** [comment]
978211
